jaxrs-tck/src/main/java/jakarta/ws/rs/tck/uribuilder/UriBuilderIT.java

```diff
@@ -61,18 +62,55 @@ public final void shouldBuildValidInstanceFromScratch()
     }
 
     /**
-     * Verifies that no invalid URI can be created from scratch.
+     * Verifies that {@code UriBuilder#build()} creates an empty {@code URI}
+     * no other methods are called on it. The created {@code URI} should be
+     * equivalent to {@code URI.create("")}.
      * 
      * @throws ExecutionException   if the instance didn't boot correctly
      * @throws InterruptedException if the test took much longer than usually
      *                              expected
      */
     @Test
-    public final void shouldNotBuildInvalidUriFromScratch() throws InterruptedException, ExecutionException {
+    public final void emptyUriBuilderBuildsEmptyUri() throws InterruptedException, ExecutionException {
         // given
-        final UriBuilder uriBuilder = UriBuilder.newInstance();
+        final URI uri = UriBuilder.newInstance().build();
```

Comment:
Can we please keep the BDD style in this test? "// given" should create the UriBuilder, "// when" should just do "build()". That would be nice. :-)